library/systemd/test/yast2/systemd_service_test.rb

```diff
@@ -48,36 +46,36 @@ module Yast
       let(:apparmor_double) { double("Service", name: "apparmor") }
       let(:cups_double) { double("Service", name: "cups") }
       let(:systemctl_stdout) do
-        File.read(File.join(__dir__, "data", "apparmor_and_cups_properties"))
+        File.read(File.join(File.expand_path("..", __dir__), "data", "apparmor_and_cups_properties"))
```

Comment:
For this I think it is better to define a data path in `test_helper.rb` and then use it here.

In `test_helper.rb`:
`DATA_PATH = File.expand_path("../data", __FILE__)`

In test:
`File.read(File.join(DATA_PATH, "apparmor_and_cups_properties")`